### PR TITLE
Fix Error Classes when loading SWEET #84

### PR DESCRIPTION
**Pull request by ignazio1977**
Some errors due to cyclic dependencies in relaMath imports closure and
OWLAPI bugs. Workaround is to add the missing declarations.

Other punning errors due to typing some properties as annotation
properties, possibly a leftover of early OWLAPI bugs or missing
dependencies when the ontologies were first created. Amended by ensuring
properties always have the same type in all their declarations.

### CODE REVIEW

File: src/stateSpectralLine.ttl [hunk 1]
```diff
@@ -31,14 +31,18 @@
 sorelch:hasSubstance rdf:type owl:AnnotationProperty .
```
Comment (smrgeoinfo) [inline src/stateSpectralLine.ttl]: sorelch:hasSubstance is defined as an object property in line 46 of  src/statePhysical.ttl (above)

Comment (ignazio1977) [reply src/stateSpectralLine.ttl]: Good catch, OWLAPI didn't find a pun there, maybe neither ontology imports the other.

**Comment by lewismc** [comment]
I can confirm that this PR removes the Error's when I load resources locally. I ran FaCT++ and  dumping out the inferred axioms right now. I'll update here with outcome. I'll also go ahead and run robot once that's done.

**Comment by ignazio1977** [comment]
I've found another forty or fifty thanks to @smrgeoinfo review, I'll update the PR accordingly. I'll also add a check in my testcase to see if there's anyother that can be spotted automatically.

**Comment by cmungall** [comment]
Thank you @ignazio1977!!!

**Comment by lewismc** [comment]
Any comments here folks?

**Comment by lewismc** [comment]
@ignazio1977 some persisting instances of punning
```
Illegal redeclarations of entities: reuse of entity http://sweetontology.net/relaSci/rangeOf in punning not allowed [ObjectProperty: rangeOf, DataProperty: rangeOf]
Illegal redeclarations of entities: reuse of entity http://sweetontology.net/relaSci/hasUpperQuantity in punning not allowed [DataProperty: hasUpperQuantity, ObjectProperty: hasUpperQuantity]
Illegal redeclarations of entities: reuse of entity http://sweetontology.net/relaSci/hasLowerQuantity in punning not allowed [DataProperty: hasLowerQuantity, ObjectProperty: hasLowerQuantity]
Illegal redeclarations of entities: reuse of entity http://sweetontology.net/relaSci/hasQuantity in punning not allowed [DataProperty: hasQuantity, ObjectProperty: hasQuantity]
```

**Comment by ignazio1977** [comment]
Yes, these are the ones I've mentioned on the latest open issue, these are not accidental duplicate declarations but actual punning in the ontology which I cannot resolve. A domain expert needs to take a look and decide whether these should be data or object properties.

**Comment by graybeal** [comment]
Is there a trivial way to see the corresponding usage of those properties? (That is, have you already made a separate document listing them?)  Or should I just use Protege?

**Comment by ignazio1977** [comment]
I'll do a search and let you know.

**Comment by ignazio1977** [comment]
From looking at the usage, these properties are referred to as data properties only inside the same ontology, and their range is set to a class not to a datatype or data range, so I'd conclude they are just mistyped in the ontology and can be easily rectified. I'll add a commit.

The warning about domain changing can maybe be fixed, but given that the ranges are not anonymous it's not a serious warning.

**Comment by lewismc** [comment]
Fantastic @ignazio1977 thank you. You should have received a link to join the committers roster as well :)

**Pull request merged**